packages/server/src/appMigrations/index.ts

```diff
@@ -10,22 +10,36 @@ export * from "./appMigrationMetadata"
 export type AppMigration = {
   id: string
   func: () => Promise<void>
+  // disabled so that by default all migrations listed are enabled
+  disabled?: boolean
 }
 
-export const getLatestMigrationId = () =>
-  MIGRATIONS.map(m => m.id)
+export function getLatestEnabledMigrationId() {
```

Comment:
It's not going to a problem at the moment but long-term we'll _probably_ want to avoid creating a copy of the migration list for every request.

Reply:
In this case it isn't copying the whole migration anymore - it creates an array of references to the enabled migrations (not cloned) and then maps out the IDs of them, theres never a full copy of the migrations.